internal/attestation/measurements/measurements.go

```diff
@@ -43,6 +43,10 @@ const (
 	// The value used to extend is derived from Constellation's master key.
 	// TODO: move to stable, non-debug PCR before use.
 	PCRIndexOwnerID = tpmutil.Handle(16)
+
+	// RTMRIndexClusterID is the RTMR we extend to mark the node as initialized.
+	// The value is the index of the RTMR + 1, since index 0 of the TDX measurements is reserved for MRTD.
+	RTMRIndexClusterID = 3
```

Comment:
The RTMR + 1 one seems like it will end up confusion later on.
Maybe I would remove RTMR here to remove and call this `TDXAttestationIndexClusterID` to disassociate that this could be an RTMR index?